Backend/Remora.Discord.API.Abstractions/API/Gateway/Events/Messages/IMessageUpdate.cs

```diff
@@ -48,8 +48,11 @@ public interface IMessageUpdate : IPartialMessage, IGatewayEvent
     /// <summary>
     /// Gets a list of users mentioned in the message.
     /// </summary>
-    new Optional<IReadOnlyList<IUserMention>> Mentions { get; }
+    new IReadOnlyList<IUserMention> Mentions { get; }
```

Comment:
We should be able to drop this (and the explicit implementations below) outright as the forwarding implementation for IPartialMessage is already handled by IMessage. We're already breaking the interface, so we might as well clean things up.